Makefile

```diff
@@ -254,6 +254,35 @@ run:
 		-h mail.my-domain.com -t $(NAME)
 	sleep 15
 
+NON_DEFAULT_DOCKER_MAIL_NETWORK_NAME=non-default-docker-mail-network
+
+test-connected-networks:
```

Comment:
I would not create a separated code for this test.

Reply:
I agree, this was a WIP state that I wanted to push to show the problem. I wanted to avoid running the whole test suite just to run this single new test. In the final version I wanted to append that to the rest.